### PR TITLE
Issue #45 - Seed.__init__ should raise a warning when the seed is lon…

### PR DESCRIPTION
**Pull request by johngrantuk**

### CODE REVIEW

File: iota/crypto/types.py [hunk 1]
```diff
@@ -75,6 +77,13 @@ class Seed(TryteString):
   References:
     - https://forum.iota.org/t/why-arent-seeds-longer-than-81-trytes-more-secure/1278
   """
+  def __init__(self, trytes=None):
+    # type: (Optional[TrytesCompatible]) -> None
+    if trytes:
+        if len(trytes) > 81:
+         warnings.warn("Beyond 81 trytes, you can't increase the security of your seed, see - https://forum.iota.org/t/why-arent-seeds-longer-than-81-trytes-more-secure/1278")
```
Comment (todofixthis) [inline iota/crypto/types.py]: Please make the indenting consistent — 2 spaces per level.

Also, the PyOTA codebase generally conforms to [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), in this case, max line length s/b 79 chars.

These changes make the code formatting more consistent, and the overall width of the code is narrower, which reduces [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load).

**Comment by todofixthis** [comment]
Hey @johngrantuk now that PyOTA v2 is out, I'd like to get this guy out the door for the 2.0.1 release.  Is there anything I can help with?

**Comment by johngrantuk** [comment]
Hi, sorry for the lack of progress, I've had limited free time unfortunately. I hope to get a bit of time on Sunday but can't guarantee it. Won't be offended if someone else gets there before me!

**Comment by todofixthis** [comment]
Hey @johngrantuk looks like somebody snuck in and took care of #45 .  I have to close this PR; sorry!

I really appreciate your contributions to PyOTA, and I hope to see you again here soon (:

**Pull request closed without merging**